src/v2/cookbook/serverless-blog.md

```diff
@@ -0,0 +1,289 @@
+---
+title: Create a CMS-Powered Blog
+type: cookbook
+order: 5
+---
+
+# Create a serverless CMS-Powered Blog Using Vue.js
+
+So you've just launched your Vue.js website, congrats! Now you want to add a blog that quickly plugs into your website and you don't want to have to spin up a whole server just to host a Wordpress instance (or any DB-powered CMS for that matter). You want to just be able to add a few Vue.js blog components and some routes and have it all just work, right? What you're looking for a "serverless" blog that's powered entirely by API's you can consume directly from your Vue.js application. This tutorial will teach you how to do just that, let's dive in!
+
+We're going to quickly build a serverless CMS-powered blog with Vue.js. It uses [ButterCMS](https://buttercms.com/), an API-first CMS that lets you manage content using the ButterCMS dashboard and integrate our content API into your Vue.js app. You can use ButterCMS for new or existing Vue.js projects.
```

Comment:
You say Serverless here and at the end without any reference or explanation to what is Serverless about it, and why that's important, or how it works. I would either remove that descriptor or spend more time explaining Serverless concepts and why that matters here. 

In my opinion, it has little to do with the rest of the article, so I'd just remove the reference for clarity and flow, but it's up to you.